.github/workflows/drone.sh

```diff
@@ -0,0 +1,54 @@
+apt update
+apt install -y python3-sphinx cmake curl jq wget
+cmake --version
+make --version
+gcc --version
+cd ../..
+curl -sL https://api.github.com/repos/flang-compiler/llvm/actions/workflows/build_llvm.yml/runs --output runs_llvm.json
+curl -sL https://api.github.com/repos/flang-compiler/flang-driver/actions/workflows/build_flang-driver.yml/runs --output runs_flang-driver.json
+wget --output-document artifacts_llvm `jq -r '.workflow_runs[0].artifacts_url?' runs_llvm.json`
+wget --output-document artifacts_flang-driver `jq -r '.workflow_runs[0].artifacts_url?' runs_flang-driver.json`
+echo "cat artifacts_llvm"
+cat artifacts_llvm
+i="0"
+while [ `jq -r '.total_count?' artifacts_llvm` == "0" ] && [ $i -lt 3 ]
+do
+    echo "No artifacts in build $i, counting from latest" 
+    i=$[$i+1] 
+    wget --output-document artifacts_llvm `jq -r --argjson i "$i" '.workflow_runs[$i].artifacts_url?' runs_llvm.json` 
+    echo "cat artifacts_llvm" 
+    cat artifacts_llvm 
+done
+echo "cat artifacts_flang-driver"
+cat artifacts_flang-driver
+i="0"
+while [ `jq -r '.total_count?' artifacts_flang-driver` == "0" ] && [ $i -lt 3 ]
```

Comment:
@xoviat it means there's an error somewhere in this line. I expect something either evaluates to an empty string or something that's expected to be numeric isn't.

I suggest you add `set -x` before this line to show what it evaluates to